modules/openapi-generator/src/main/resources/C-libcurl/api-body.mustache

```diff
@@ -325,11 +325,6 @@ end:
                     localVarBodyParameters,
                     "{{{httpMethod}}}");
 
-    {{#responses}}
```

Comment:
This code has already been changed by https://github.com/OpenAPITools/openapi-generator/pull/12056